### PR TITLE
fix(tabs): extra spacing between tab and tab content in ie 11

### PR DESCRIPTION
**Pull request by florin-borceanu**
## PR Checklist

Please check if your PR fulfills the following requirements:

* [ ] Tests for the changes have been added (for bug fixes / features)
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

* [x] Bugfix
* [ ] Feature
* [ ] Code style update (formatting, local variables)
* [ ] Refactoring (no functional changes, no api changes)
* [ ] Build related changes
* [ ] CI related changes
* [ ] Documentation content changes
* [ ] clarity.design website / infrastructure changes
* [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #3748 

## Does this PR introduce a breaking change?

* [ ] Yes
* [x] No

## Other information
Tested on:
Internet Explorer 11 ✔
Edge ✔
Google Chrome ✔
Mozilla ✔
Safari ✔

### CODE REVIEW

File: src/clr-angular/layout/tabs/tab-content.ts [hunk 1]
```diff
@@ -14,13 +14,13 @@ let nbTabContentComponents: number = 0;
...
     <ng-template #tabContentProjectedRef>
-      <section [id]="tabContentId" role="tabpanel" class="tab-content" [class.active]="active"
+      <div [id]="tabContentId" role="tabpanel" class="tab-content" [class.active]="active"
                [hidden]="!active"
                [attr.aria-labelledby]="ariaLabelledBy"
                [attr.aria-expanded]="active"
                [attr.aria-hidden]="!active">
         <ng-content></ng-content>
-      </section>
+      </div>
```
[the remark this answers is not in the document]
Comment (florin-borceanu) [reply src/clr-angular/layout/tabs/tab-content.ts]: Here is used `section`:
image

and here is used `div`:
image

Both of them are tested on Internet Explorer 11, with the same CSS. I can't understand either why `section` behave different than a `div` only on IE 11.
If you suggest, I will try to resolve this with some CSS refactorization.

**Comment by Jinnie** [comment]
Hi @florin-borceanu ,

As @Shijir mentions, this was fixing an issue with content not expanding to the component container. It was a regression, so it was not a matter of finding the best experience, but ensuring consistent behavior. Here's the PR where it was introduced (you can see the original issue linked there):
https://github.com/vmware/clarity/pull/3298

Make sure you're not breaking this case. There is a unit test that covers it, so tests should help you, but it's good to verify it manually, too. 

Thanks for your work on this!

**Comment by gnomeontherun** [comment]
@florin-borceanu what is the status here?

**Comment by florin-borceanu** [comment]
The pull request is complete. I found out that Internet Explorer renders differently the `section` from `div` element. The simplest fix is to change the `section` with a `div`, this way the fix with `display: inline-block` remains there. If you suggest, I'll try to solve it by refactoring some CSS, but this will take some time because this involves some major changes.

**Pull request merged**